src/commands/stac-catalog/stac.catalog.ts

```diff
@@ -88,7 +88,7 @@ export async function createLinks(basePath: string, templateLinks: st.StacLink[]
       const buf = await fsa.read(coll);
       const collection = JSON.parse(buf.toString()) as st.StacCollection;
       // Multihash header 0x12 - Sha256 0x20 - 32 bits of hex digest
-      const checksum = '1220' + createHash('sha256').update(buf).digest('hex');
+      const checksum = Sha256Prefix + createHash('sha256').update(buf).digest('hex');
```

Comment:
it would be better to store extract this as a function as the prefix is only valid for the combination of 'sha256' and 'hex' digest.

there is a function that does this for streams, could put another one next to it for buffers or strings.